content/docs/gke/deploy/minikf-gcp.md

```diff
@@ -0,0 +1,93 @@
++++
+title = "Deploy using MiniKF on GCP"
```

Comment:
One more comment: Please would you move this doc from its current location (`/docs/gke/deploy/minikf-gcp.md`) to the **getting-started -> workstation** section: `/docs/started/workstation/minikf-gcp.md`.

The suggested location under **getting-started -> workstation**  will help to distinguish between a Kubeflow deployment on GKE and a Kubeflow deployment on MiniKF running on Compute Engine.